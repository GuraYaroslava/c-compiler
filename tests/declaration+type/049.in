int a[10];

a + 1;

1 + a;