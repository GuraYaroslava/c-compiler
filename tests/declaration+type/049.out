table 0:
`int`
`float`
`char`

table 1:
variable `a` type of array 10 of `int`

tree 0:
         a
    + <
         1

tree 1:
         1
    + <
         a

